tests/csm/rest/test_iam_users.py

```diff
@@ -4512,3 +4512,51 @@ def test_39406(self):
             assert_utils.assert_true(resp[0], resp[1])
         self.log.info("[END]Creating IAM users with different tenant")
         self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-42270')
+    def test_42270(self):
+        """
+        Test GET IAM user with valid max_entries and marker using Admin login
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Step 1: Creating 10 IAM users.")
+        users_list = []
+        for count in range(10):
+            resp = self.csm_obj.verify_create_iam_user_rgw(verify_response=True)
+            assert_utils.assert_true(resp[0], resp[1])
+            users_list.append(resp[1]["uid"])
```

Comment:
I think you will need to add resp['tenant'] + "$" + resp['user_id'] in user_list, @kachhwahadivya pls correct me if I am wrong

Reply:
@kachhwahadivya please let me know as I'm not aware about this

Reply:
As per discussion with @kachhwahadivya  this is not reqd as tenant field will be empty here